src/completers/git.zsh

```diff
@@ -330,6 +331,67 @@ _fzf_complete_git() {
         return
     fi
 
+    if [[ $subcommand = 'push' ]]; then
+        local prefix_option completing_option
+
+        local git_options_argument_required=(--exec -o --push-option --receive-pack --recurse-submodules --repo)
+        local git_options_argument_optional=(--force-with-lease --signed)
+
+        if completing_option=$(_fzf_complete_parse_completing_option "$prefix" "$last_argument" ${(F)git_options_argument_required} ${(F)git_options_argument_optional}); then
+            if [[ $completing_option = --* ]]; then
+                prefix_option=$completing_option=
+            else
+                prefix_option=${prefix%%${completing_option[-1]}*}${completing_option[-1]}
+            fi
+            prefix=${prefix#$prefix_option}
+        fi
+
+        local prefix_ref=${prefix%[^:]#}
+
+        case $completing_option in
+            --signed)
+                local signed=(false if-asked true)
+                _fzf_complete_git_constants '' "${(F)signed}" $@
+                ;;
+
+            -o|--push-option)
+                ;;
+
+            --exec|--receive-pack)
+                ;;
+
+            --force-with-lease)
+                prefix=${prefix#*:} _fzf_complete_git-commits '' $@
+                ;;
+
+            --repo)
+                _fzf_complete_git-remotes '' $@
+                ;;
+
+            --recurse-submodules)
+                local recurse_submodules=(check no on-demand only)
+                _fzf_complete_git_constants '' "${(F)recurse_submodules}" $@
+                ;;
+
+            *)
+                local repository
+                if ! repository=$(_fzf_complete_git_parse_argument 1 "$arguments" "${(F)git_options_argument_required}") && [[ -z $repository ]]; then
+                    _fzf_complete_git-remotes '' $@
+                    return
+                fi
+
+                if [[ $prefix = *:* ]]; then
+                    prefix=${prefix#*:} repository=$repository _fzf_complete_git-refs '' $@
```

Comment:
```suggestion
                    prefix=${prefix#*:} _fzf_complete_git-refs '' $@
```

Reply:
It is on `git pull` too, so I fix it together.
https://github.com/chitoku-k/fzf-zsh-completions/blob/ef2836f75c629eb985f18108fb5733fde55abe4c/src/completers/git.zsh#L327

Reply:
Thanks, that has originally been freaky.